packages/codemods/transforms/rename-deprecated-props-flex.js

```diff
@@ -1,10 +1,8 @@
-import get from 'lodash.get'
-
 const update = (path) => {
   const Node = path.value
 
   // Dumb way to skip text elements since they also can have an align prop
-  if (get(Node, 'openingElement.name.name', '').toLowerCase().includes('text')) {
+  if (Node?.openingElement?.name?.name?.toLowerCase()?.includes('text')) {
```

Comment:
(super-n-b) If that last `name` returns, `toLowerCase()` should work, no?
```suggestion
  if (Node?.openingElement?.name?.name?.toLowerCase().includes('text')) {
```